src/views/v2/components/fields/radio.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/components/fields/radio.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Radio
+ */
+
+$field      = $this->get( 'field' );
+$field_name = tribe_tickets_ar_field_name( $ticket->ID, $field->slug );
+$required   = tribe_tickets_ar_field_is_required( $field );
```

Comment:
Would love to chat about the creation of these template tags, I wonder if they could be inside of a Template class that we can use here via a variable, to avoid the notion of this being global, since it's specific to this template system.

Reply:
the field name and id are particular to a field meta, so they have to be related to that specifically. With these template tags I abstracted things that we were using on the templates over and over again, and that was (in the past) giving some bugs and problems related to inconsistencies.